### PR TITLE
chore(qa): Upgrade to Codeceptjs 3.0.1

### PR DESCRIPTION
**Pull request by themarcelor**
⚠️ This is a big change so any additional review/scrutiny is deeply appreciated.

**Upgrading to CodeceptJS 3** by following the instructions in the official wiki:
https://github.com/codeceptjs/CodeceptJS/wiki/Upgrading-to-CodeceptJS-3

Michael Bodnarchuk's Slack post announcing the Release Candidate (RC) back in September 23rd:
https://codeceptjs.slack.com/archives/C94G8JXJB/p1600882274026500

### New Features
- https://github.com/codeceptjs/CodeceptJS/blob/codeceptjs-v3.0/CHANGELOG.md

### Breaking Changes
- 1️⃣ Syntax Change
CodeceptJS 3.0 introduced a new syntax for declaring tests, instead of:
```
Scenario('title', (I, loginPage) => {})
```
they use a new way of passing arguments into a test, via destruction:
```
Scenario('title', ({ I, loginPage }) => {})
```

### Improvements

- New `toTry()` function that returns true/false for a given Selenium operation.

### Dependency updates

- New change in `package.json` to set:

```
    "codeceptjs": "^3.0.0-beta.3",
```

## Making sure all test suites work

- [x] test-apis-dataUploadTest
- [x] test-google-googleServiceAccountRemovalTest
- [x] test-portal-indexingPageTest 
- [x] test-apis-linkGoogleAccountTest
- [x] test-apis-metadataIngestionTest
- [x] test-portal-dataUploadTest
- [x] test-google-googleServiceAccountTest
- [x] test-google-googleServiceAccountKeyTest

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "chai-subset": "^1.6.0",
-    "codeceptjs": "^2.6.10",
+    "codeceptjs": "^3.0.0-beta.3",
```
Comment (frickjack) [inline package.json:26]: do you want to wait for it to go GA?

**Review by frickjack — APPROVED** [review]
looks ok to me - except maybe you want to wait for the thing to come out of beta?

**Review by haraprasadj — APPROVED** [review]
Maybe safe to have at least one run of all the tests together before merging

File: package.json (hunk 1 above)
Comment (themarcelor) [reply package.json]: Alright @frickjack , looks like they published `3.0.1` 😄 So, I will run a full set of tests, merge and keep a close eye on the pipeline.

**Review by jingh8 — APPROVED** [review]

**Pull request merged**